src/adv.py

```diff
@@ -33,19 +36,62 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
-#
-# Main
-#
-
-# Make a new player object that is currently in the 'outside' room.
-
-# Write a loop that:
-#
-# * Prints the current room name
-# * Prints the current description (the textwrap module might be useful here).
-# * Waits for user input and decides what to do.
-#
-# If the user enters a cardinal direction, attempt to move to the room there.
-# Print an error message if the movement isn't allowed.
-#
-# If the user enters "q", quit the game.
+# ======================= Add items to rooms ========================
+
+room['outside'].items.append(Item("rock", "It's a fist sized rock"))
+room['outside'].items.append(Item("debris", "A pile of trash"))
+room['foyer'].items.append(
+    Item("orb", "A funny looking key with a skull on it"))
+room['overlook'].items.append(
+    Item("key", "A funny looking key with a skull on it"))
+
+# print("This:")
+# print(room['outside'].items[0].description)
+# print(room['outside'].items[1])
+
+# =========================  Main  ================================
+
+# print(room['outside'].items)
+
+player_name = input("What's your character's name?")
+player = Player(player_name, room['outside'])
+print(f"\nWelcome {player}\n")
+
+
+while True:
+    # * Prints the current description (the textwrap module might be useful here).
+    print("------------------------------\n")
+    print(player.current_room)
+    print(f"You can go: {player.current_room.get_room_exits()}\n")
+    print(f"You see:")
+    for item in player.current_room.items:
+        print(f" - {item}")
+    cmd = input("\n=>")
+    print("")
+
+    if cmd == "q":   # If the user enters "q", quit the game.
+        break
+
+    elif cmd in ["n", "s", "e", "w"]:           # Directional commands
```

Comment:
consider making this part of a method in the player class; it will keep the main loop cleaner and easier to read and manage.